docs/index.md

```diff
@@ -19,10 +19,9 @@ Axual Provider allows using Axual's Self-Service for Apache Kafka functionality
 	- To secure which applications are authorised to access streams, we support
```

Comment:
The docs needs to clearly have written that we do not support doing changes to resources managed by Terraform from UI as instructed by @xsreality

Reply:
Drift is inherently supported in Terraform. The user is responsible for handling this.

Reply:
Added to index.md